js/main.js

```diff
@@ -1,28 +1,108 @@
 /* global data */
 /* exported data */
-var $photoURL = document.querySelector('.photourl');
-var $preview = document.querySelector('.preview');
-
-function updateImg(event) {
-  $preview.setAttribute('src', $photoURL.value);
+var $image = document.querySelector('#image');
+var $photoUrl = document.querySelector('#photourl');
+function updateImagePreview(event) {
+  $image.setAttribute('src', $photoUrl.value);
 }
-
-$photoURL.addEventListener('input', updateImg);
-
+$photoUrl.addEventListener('input', updateImagePreview);
 var $form = document.querySelector('form');
-
 function updateEntry(event) {
   event.preventDefault();
-  var entriesObj = {
+  var entryObj = {
     title: $form.elements.title.value,
     photourl: $form.elements.photourl.value,
     notes: $form.elements.notes.value,
-    entryId: data.nextEntryId
+    nextEntryId: data.nextEntryId
```

Comment:
This property should be named `entryId`, `nextEntryId` is not an accurate name to describe this entry's id.